README.md

```diff
@@ -10,18 +10,18 @@ This project has multiple versions of EnergyPlus ready for use in a single conta
 
 Below is a table of the various docker tags and their meanings as seen on [this page](https://hub.docker.com/r/nrel/energyplus/tags/).
 
-| Tag     | Description                                                                             |
-|---------|-----------------------------------------------------------------------------------------|
-| x.y.z   | Build of official EnergyPlus release (recommended use)                                  |
-| latest  | Latest official release of EnergyPlus (e.g. 2.5.1)                                      |
-| develop | Release of [develop branch](https://github.com/NREL/docker-energyplus/tree/develop)     |
+| Tag     | Description                                                                         |
+|---------|-------------------------------------------------------------------------------------|
+| x.y.z   | Build of official EnergyPlus release (recommended use)                              |
+| latest  | Latest official release of EnergyPlus (e.g. 22.1.0)                                 |
```

Comment:
so energyplus is now using YY.M.patch syntax? Love it!

Reply:
Yup, they finally switched over to Ubuntu-style versioning

Reply:
😍